Pipfile

```diff
@@ -50,6 +50,8 @@ tomte = {version = "==0.2.2", extras = ["tox", "tests"]}
 apduboy = ">=0.5.0"
 docspec = "==2.0.2"
 docspec-python = "==2.0.2"
+hexbytes = "==0.3.0"
+flashbots = "==1.1.1"
```

Comment:
Same for this, it will fail for py <3.9

Reply:
Addressed in 46191c12c30857d82a85d842ac2209c1535b4a43.